source/shared/cpp/ObjectModel/Icon.h

```diff
@@ -25,13 +25,13 @@ class Icon : public BaseCardElement
     ~Icon() = default;
 
     Json::Value SerializeToJsonValue() const override;
-    
```

Comment:
nit unnecessary changes, can be removed.

Reply:
Tried reverting this earlier. Android studio is applying its formatting on the file, hence these empty spaces are getting added. Leaving it as it is.